budget/config/scipooldev.yaml

```diff
@@ -0,0 +1,7 @@
+# budget rules per user by synapse team id
```

Comment:
are these files in`budget/config` folder needed anymore?  I don't see them used anywhere.